### DISCUSSION TITLE
could i upload the jpeg stream to the kinesis video server?

### DISCUSSION THREAD

**Discussion by pppaulpeter** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1551] [Q&A] [answered]
i have already porting the libary to my camera, but it seems just support upload the H264 raw frame to the cloud. may i know is that possible to upload jpeg frame to the cloud?

**Comment by niyatim23** [comment, marked as answer]
Unfortunately jpeg is not one of the supported formats. A list of supported formats can be found [here](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/tree/master/src/source/Rtp/Codecs)
